include/oboe/AudioStream.h

```diff
@@ -19,6 +19,8 @@
 
 #include <cstdint>
 #include <ctime>
+#include <atomic>
```

Comment:
Nit: Move <atomic> above <cstdint>. Alphabetical order for grouped includes is ANdroid style.